dt/uni04delta/kustomization.yaml

```diff
@@ -17,9 +17,9 @@ transformers:
         create: true
 
 components:
-  - ../../lib/networking/metallb
-  - ../../lib/networking/netconfig
-  - ../../lib/networking/nad
+  - networking/metallb
```

Comment:
I'm wondering (but I don't have probably enough knowledge) if, instead of duplicating the components we used to have in `lib/` there's a way to extend them as we've done w/ `storagemgmt` for Ceph.
The nfs use case is a similar example.
The only use case where we need that network in the ctlplane is `test-operator` so far, while it's required on the edpm nodes.

Reply:
Ok, while I was looking at the equivalent RGW/Swift change, I found an already started conversation around this topic  [1]. No need to duplicate answers.

[1] https://github.com/openstack-k8s-operators/architecture/pull/386#issuecomment-2340198597

Reply:
removed